Sources/UXKit/AppKit/NSTableViewCell.swift

```diff
@@ -21,7 +21,7 @@
    *
    * NOTE: This one doesn't actually support an image :-/
    */
-  open class NSTableViewCell : NSTableCellView {
+  open class NSTableViewCell : NSTableCellView, NSTextFieldDelegate {
```

Comment:
I'm not sure what to make of that (and the related changes below, including the 1st responder changes). The behaviour added seems specific to a certain use-case, not required for UIKit/AppKit compatibility?
Can you elaborate why we need that in UXKit?